### DISCUSSION TITLE
Trezor with multiple accounts

### DISCUSSION THREAD

**Discussion by freddiemacd** [cryptoadvance/specter-desktop#1099] [Hardware Wallets]
I have a trezor with addresses used in three levels of accounts. Only the account in the first level shows up. How to I add the other accounts?

To be clear, these are all using the same seed words without using different passphrases.

**Comment by ben-kaufman** [comment]
Hi, you need to add the xpubs for each account, then create a wallet with each account's xpub.
To add the xpub, click on the device page, then "Add more keys" at the bottom, then in the table click "Edit" on the top right of the table and a row will show at the bottom, click there "Add Account", specify account number and click add. Then click get over USB to get the keys and then add keys to save them. This will let you create a wallet with them. Create the wallet like the first one, just on the page where you can name thee wallet, select the right account key in the list at the bottom.

**Reply by freddiemacd** [reply]
That works.
Thanks!

**Reply by JuanOfTheNorth** [reply]
If my first account is Segwit, and my 2nd account is taproot, should I still be able to accomplish the above? 

I added keys, then created a customized one with Purpose #1 Single Sig (Taproot) & Derivation m/86h/0h/1h

I then choose add new wallet but it only shows 
#0 Single Sig (Segwit) 
#1 Single Sig (Segwit)
